services/example-laserfocusworld/server/templates/content/index.marko

```diff
@@ -83,6 +83,13 @@ $ const displayPublishedDate = ["event", "webinar", "contact"].includes(type) ?
                   <@link class="btn btn-lg btn-primary" />
                 </default-theme-content-download>
               </if>
+              <if(type === 'webinar')>
```

Comment:
Change quote style to `"`...

Also, do you want to apply this change to UP as well.